app/plugins/statisticsViewer/statisticsViewerPlugin.php

```diff
@@ -74,6 +74,7 @@ public function hookRenderMenuBar($pa_menu_bar) {
 				foreach ($va_universes as $key => $value) {
 					$va_menu_items[$value] = array(
 						'displayName' => _t($value),
+						'requires' => array('action:can_use_statistics_viewer_plugin' => 'AND'),
```

Comment:
Only admins can currently access the statistics viewer even though there is an action permission listed under user roles